tests/playwright/resources/bootable-containerfile

```diff
@@ -0,0 +1,4 @@
+FROM quay.io/centos-bootc/fedora-bootc:eln
```

Comment:
I think I commented somewhere else, it's missing the Copyright/header and the newline at the bottom

Reply:
We are missing headers in podman-desktop, might be there.